pkg/command/root_test.go

```diff
@@ -1300,21 +1306,22 @@ func TestCommandRemapping(t *testing.T) {
 			expected:                []string{"dummy2 commands"},
 			allowActiveContextCheck: true,
 		},
-		{
-			test: "mapped plugin is only one for target, top level should show target",
-			pluginVariants: []fakePluginRemapAttributes{
-				{
-					name:      "dummy2",
-					target:    configtypes.TargetK8s,
-					invokedAs: []string{"dummy"},
-					aliases:   []string{"dum"},
-				},
-			},
-			args:     []string{""},
-			expected: []string{"kubernetes"},
-		},
-		{
-			test: "mapping plugins when conditional on active contexts",
+		// Only works for the Kubernetes target which is now always hidden
```

Comment:
nit: not sure if you meant the phrasing in the comment. Perhaps you meant something like "only works for the operations target since kubernetes target is always hidden?"

Reply:
sgtm